target_clickhouse/sinks.py

```diff
@@ -177,7 +177,8 @@ def _validate_and_parse(self, record: dict) -> dict:
         record = pre_validate_for_string_type(record, self.schema, self.logger)
 
         try:
-            self._validator.validate(record)
+            if self.validate_schema:
+                self._validator.validate(record)
```

Comment:
Would it make sense to convert this whole try-except block into `return super()._validate_and_parse(record)`?

Reply:
Ah yes, only now I have time look at the parent method, looks identical. I was in rush to fix on prod yesterday so didn't read everything. Thank @edgarrmondragon for the suggestion.